frontend/src/static/js/main.js

```diff
@@ -15,6 +15,55 @@ window.addHash = function addHash(newKey, newVal) {
   window.location.hash = hash.join('&');
 };
 
+const DRAG_BAR_WIDTH = 13.25;
+const SCROLL_BAR_WIDTH = 17;
+const GUIDE_BAR_WIDTH = 2;
+
+const throttledEvent = (delay, handler) => {
+  let lastCalled = 0;
+  return (...args) => {
+    if (Date.now() - lastCalled > delay) {
+      lastCalled = Date.now();
+      handler(...args);
+    }
+  };
+};
+
+let mouseMove = () => {};
+let guideWidth = (0.5 * window.innerWidth - (GUIDE_BAR_WIDTH / 2))
+    / window.innerWidth;
+let flexWidth = 0.5;
+
+const registerMouseMove = () => {
+  const innerMouseMove = (event) => {
+    guideWidth = (
+        Math.min(
+            Math.max(
+                window.innerWidth - event.clientX,
+                SCROLL_BAR_WIDTH + DRAG_BAR_WIDTH,
+            ),
+            window.innerWidth - SCROLL_BAR_WIDTH,
+        )
+        - (GUIDE_BAR_WIDTH / 2)
+    ) / window.innerWidth;
+    window.$('tab-resize-guide').style.right = `${guideWidth * 100}%`;
+  };
+  window.$('tab-resize-guide').style.display = 'block';
+  window.$('app-wrapper').style['user-select'] = 'none';
+  mouseMove = throttledEvent(30, innerMouseMove);
+};
+
+const deregisterMouseMove = () => {
```

Comment:
lint your code, migth not be able to do this, do `window.deregisterMouseMove` instead, same above

Reply:
there is this pesky #514 that is going to go in, it will fail your code, so lint on local first

Reply:
I changed `const xxx = ...` to `window.xxx =  ...` but I don't lint everything